worker.config.json

```diff
@@ -5,5 +5,10 @@
         "defaultExecutablePath":"node",
         "defaultWorkerPath":"dist/src/nodejsWorker.js",
         "workerIndexing": "true"
+    },
+    "processOptions": {
+        "initializationTimeout": "00:05:00",
+        "environmentReloadTimeout": "00:05:00",
+        "processShutdownTimeout": "00:01:00"
```

Comment:
I worry increasing the shutdown timeout could have unknown ripple effects. Worker timeout is still a relatively new feature and and I generally assume shutdown is supposed to happen fast. I don't think we should change this unless people ask for it

Reply:
I was hesitant about making this change too, so I'm on board with scrapping it. That being said, we'd have to advertise that `appTerminate` hooks must terminate within 10 seconds or less. Which isn't a tall ask, but I think it should be made clear